### DISCUSSION TITLE
What version should I be running

### DISCUSSION THREAD

**Discussion by bruno-j-nicoletti** [foonathan/lexy#218] [General]
Hi,

The most recent release tag is [v2022.12.1](https://github.com/foonathan/lexy/releases/tag/v2022.12.1) which I am currently working with. Is there or will there be a more recent release tag I should work from?

Thanks

Bruno

**Comment by foonathan** [comment]
Hi, I plan on tagging a new release over the holidays. But there is no harm in living at head ;)

**Reply by bruno-j-nicoletti** [reply]
Thanks. I'm now pulling from the head of main. 

FWIW, there was a bug in v2022.12.1 that breaks in clang 19 and c++26 but is fixed on main.

https://github.com/foonathan/lexy/blob/f68737b725116d00e5582602e22604a14fc26547/include/lexy/input_location.hpp#L165

`rhs._colum_nr` is a typo, it should be `rhs._column_nr`
